src/test/java/be/seeseemelk/mockbukkit/ServerMockTest.java

```diff
@@ -183,6 +183,24 @@ void getBukkitVersion_NotNull()
 		assertNotNull(server.getBukkitVersion());
 	}
 
+	@Test
+	void getBukkitVersion_CorrectPattern()
+	{
+		assertTrue(server.getBukkitVersion().matches("1\\.[0-9]+(\\.[0-9]+)?-R[0-9].[0-9]-SNAPSHOT"));
```

Comment:
According to https://www.regexplained.co.uk/ This doesn't seem to match for Dots

Reply:
It does match otherwise the test would not pass. If you copy it directly from the string you have to remove the extra backslash before the dots when putting it into a regex validator.